alpaca/broker/client.py

```diff
@@ -460,6 +458,41 @@ def _get_account_activities_iterator(
             # set the pake token to the id of the last activity so we can get the next page
             request_fields["page_token"] = last_result["id"]
 
+    @staticmethod
+    def _validate_pagination(
+        max_items_limit: Optional[int], handle_pagination: Optional[PaginationType]
```

Comment:
This and `_return_paginated_result` should probably live on the RESTClient base class

Reply:
Refactored!